packages/ocaml-option-spacetime/ocaml-option-spacetime.1/opam

```diff
@@ -0,0 +1,8 @@
+opam-version: "2.0"
+synopsis: "Set OCaml to be compiled with spacetime activated"
+depends: [
+  # Not available until options are ported to earlier compilers
+  "ocaml-variants" {post & >= "4.12.0~" & < "4.12"}
```

Comment:
The `>= "4.12.0~"` is here for all `option` packages at the moment, since earlier `ocaml-variants` packages don't support them.
This prevents you from installing `ocaml-variants.4.11.1+*` together with this package and expecting it do to anything.

Reply:
Ah, I see!